server/sonar-server/src/main/java/org/sonar/server/component/index/ComponentIndexQuery.java

```diff
@@ -29,12 +29,18 @@
 
 public class ComponentIndexQuery {
 
+  public enum Sort {
+    NAME, SCORE
+  }
```

Comment:
I also have a slight preference to call the constants `Sort.BY_ASCENDING_NAME` and `Sort.BY_SCORE`.